CTFd/__init__.py

```diff
@@ -100,35 +100,31 @@ def _load_template(self, name, globals):
 
 
 class ThemeLoader(FileSystemLoader):
-    """Custom FileSystemLoader that switches themes based on the configuration value"""
-
-    def __init__(self, searchpath, encoding="utf-8", followlinks=False):
+    """Custom FileSystemLoader that is aware of theme structure and config.
+    """
+
+    DEFAULT_THEMES_PATH = os.path.join(os.path.dirname(__file__), "themes")
+    _ADMIN_THEME_PREFIX = ADMIN_THEME + "/"
```

Comment:
Is there a reason that this variable is "private"?

Reply:
Mostly to discourage external use since it's relevant only to the `ThemeLoader` and nobody else should really be talking about filesystem paths for themes. Happy to make it public though since it's barely relevant.